.jenkins/Jenkinsfile

```diff
@@ -32,7 +50,25 @@ def simulationTest(String compiler, String unit, String suite ) {
             docker.withRegistry(OETOOLS_REPO, OETOOLS_REPO_CREDENTIAL_ID) {
                 docker.image(OETOOLS_IMAGE).inside {
                     timeout(15) {
-                        sh "./scripts/test-build-config -p $unit -b $suite --compiler=$compiler"
+                        dir('build') {
+                            sh """
+                            if [ "${compiler}" == "gcc" ]; then
+                                export CC="gcc"
+                                export CXX="g++"
+                            elif [ "${compiler}" == "clang-7" ]; then
+                                export CC="clang-7"
+                                export CXX="clang++-7"
+                            fi
+
+                            if [ "${unit}" == "SGX1FLC" ]; then
+                                cmake .. -DCMAKE_BUILD_TYPE=${suite} -DUSE_LIBSGX=1
+                            else
+                                cmake .. -DCMAKE_BUILD_TYPE=${suite} 
+                            fi
+                            make
+                            OE_SIMULATION=1 ctest --output-on-failure
```

Comment:
This I like! Much better than explicitly _not_ passing `-d` 😉

Reply:
I found `-d` to be quite confusing 😄